src/screens/Inventory/AddInventory.tsx

```diff
@@ -93,11 +90,4 @@ function AddInventory(props: AddInventoryProps) {
   );
 }
 
-const mapStateToProps = (state: RootState) => ({
```

Comment:
I see we're removing `mapStateToProps` in favor of `useSelector`. I'm fine with this, but is there a general rule where we'd want to use `mapStateToProps` over `useSelector`? Or do we want to just migrate to `useSelector` longterm (assuming we're using functional components)

Reply:
Yeah I think in general I ended up moving all of them over to use selectors because I was always grabbing the same kinds of info, which made it a lot cleaner and more readable to switch to selectors. Also, it got pretty nasty with all the nested params I needed for typescript to agree with `mapStateToProps` (eg `const mapStateToProps = (state: RootState, ownProps: { location: { state: { purchaseRequest: { inventoryId: string; }; }; }; }) => ({` from [here](https://github.com/calblueprint/meepanyar/pull/60/files/afe73e075a92a0ef25939bb4b0eb687b55748af2#diff-bd8ec877cd98f9458d9ae9715e5c346b6a69661fc3cd5671b522b40f1f4ad27fR61-R63).

In general, I think the Redux docs do a better job explaining this, but they did mention

> The existing connect API still works and will continue to be supported, but the hooks API is simpler and works better with TypeScript. ([Source](https://react-redux.js.org/api/hooks#hooks))